app/konnect/gateway-manager/kic.md

```diff
@@ -54,6 +54,13 @@ Item | Description
 **Summary** | This section displays the traffic and error rate of your KIC data plane node.  
 **Analytics** | Analytics data for the KIC data plane node. You can configure the analytics options using the [**Analytics tool**](/konnect/analytics/). 
 
+{% if_version gte:3.3.x %}
```

Comment:
```suggestion
```
I didn't realize this was a konnect doc 🤦‍♀️ . Doesn't need or accept version tags.